suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -418,3 +423,14 @@ export const initDevices = createThunk(
         }
     },
 );
+
+export const createImportedAccountThunk = createThunk(
```

Comment:
This doesn't seems to create any account but only device, please rename it to reflect that.

Reply:
https://github.com/trezor/trezor-suite/pull/9694/commits/c9f9b0587b4ba3bea0645ae010e508e0e0911fc4